test/strongbox/catalogue_test.clj

```diff
@@ -272,7 +272,17 @@
           expected {:download-url "https://edge.forgecdn.net/files/3104/62/Pawn-2.4.5.zip",
                     :interface-version 90000,
                     :version "2.4.5"
-                    :game-track :retail}
+                    :game-track :retail
+                    :release-list [{:download-url "https://edge.forgecdn.net/files/3104/62/Pawn-2.4.5.zip",
+                                    :game-track :retail,
+                                    :interface-version 90000,
+                                    :release-label "[WoW 9.0.1] Pawn-2.4.5.zip",
+                                    :version "2.4.5"}
+                                   {:download-url "https://edge.forgecdn.net/files/3104/062/Pawn-2.4.5.zip",
+                                    :game-track :retail,
+                                    :interface-version 90000,
+                                    :release-label "[WoW 9.0.1] Pawn-2.4.5.zip",
+                                    :version "Pawn-2.4.5.zip"}]}
```

Comment:
we have two almost identical fixtures because of the way curseforge list their releases.

we should remove releases from the game-version-release-list that have identical URLs from the latest-game-files list.

Reply:
also, that release-label has a filename extension that could be stripped. only affects non-identical versions